src/estimagic/estimation/estimate_ml.py

```diff
@@ -389,40 +399,28 @@ def _get_free_cov(
         seed,
     ):
         if method not in self._valid_methods:
-            raise ValueError()
-
-        internal_jac = self._internal_jacobian
-        internal_hess = self._internal_hessian
-        design_info = self._design_info
-        converter = self._converter
-        flat_params = self._flat_params
-
-        if method == "jacobian":
-            int_cov = cov_jacobian(internal_jac)
-        elif method == "hessian":
-            int_cov = cov_hessian(internal_hess)
-        elif method == "robust":
-            int_cov = cov_robust(jac=internal_jac, hess=internal_hess)
-        elif method == "cluster_robust":
-            int_cov = cov_cluster_robust(
-                jac=internal_jac, hess=internal_hess, design_info=design_info
-            )
-        elif method == "strata_robust":
-            int_cov = cov_strata_robust(
-                jac=internal_jac, hess=internal_hess, design_info=design_info
-            )
+            msg = f"Invalid method: {method}. Valid methods are {self._valid_methods}."
+            raise ValueError(msg)
+        seed = self._seed if seed is None else seed
```

Comment:
Instead of this, I think we should set a default seed for every method that needs a seed. If someone wants to pass `seed=None` this should be possible and actually use no seed instead of using the default seed. 

Of course, if seed is None, no result should be chached.

Reply:
Use `estimagic.config.DEFAULT_SEED`